### PR TITLE
Unit tests for domain & data, CI setup for tests

### PR DESCRIPTION
**Pull request by ALEGATOR1209**

### CODE REVIEW

File: src/main/kotlin/com/fictadvisor/pryomka/domain/models/Application.kt [hunk 1]
```diff
@@ -14,7 +14,7 @@ data class Application(
         Cancelled,
     }
 
-    operator fun plus(documents: List<DocumentType>) = copy(
+    operator fun plus(documents: Set<DocumentType>) = copy(
         documents = documents
     )
```
Comment (NeProgramist) [inline src/main/kotlin/com/fictadvisor/pryomka/domain/models/Application.kt:19]: add elements to set, not remove

File: src/test/kotlin/com/fictadvisor/pryomka/ApplicationTest.kt [hunk 2]
```diff
@@ -9,11 +9,11 @@ import kotlin.test.assertEquals
...
-        withTestApplication({ configureRouting() }) {
-            handleRequest(HttpMethod.Get, "/").apply {
-                assertEquals(HttpStatusCode.OK, response.status())
-                assertEquals("{\"Test\":\"Kek\"}", response.content)
-            }
-        }
+//        withTestApplication({ configureRouting() }) {
+//            handleRequest(HttpMethod.Get, "/").apply {
+//                assertEquals(HttpStatusCode.OK, response.status())
+//                assertEquals("{\"Test\":\"Kek\"}", response.content)
```
Comment (NeProgramist) [inline src/test/kotlin/com/fictadvisor/pryomka/ApplicationTest.kt:15]: delete unused code

File: src/test/kotlin/com/fictadvisor/pryomka/data/datasources/FsDocumentContentDataSourceTest.kt [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+        )
+
+        val ds = FsDocumentContentDataSource(secret, Dispatchers.Unconfined)
+
+        // WHEN
+        ds.save(metadata, content.byteInputStream())
+        val file = File(metadata.path.value)
+
+        // THEN
+        assertNotEquals(content, file.readText())
```
Comment (NeProgramist) [inline src/test/kotlin/com/fictadvisor/pryomka/data/datasources/FsDocumentContentDataSourceTest.kt:99]: replace with contains func

File: src/main/kotlin/com/fictadvisor/pryomka/domain/models/Application.kt (hunk 1 above)
Comment (ALEGATOR1209) [reply src/main/kotlin/com/fictadvisor/pryomka/domain/models/Application.kt]: Fixed

File: src/test/kotlin/com/fictadvisor/pryomka/ApplicationTest.kt (hunk 2 above)
Comment (ALEGATOR1209) [reply src/test/kotlin/com/fictadvisor/pryomka/ApplicationTest.kt]: fixed

File: src/test/kotlin/com/fictadvisor/pryomka/data/datasources/FsDocumentContentDataSourceTest.kt (hunk 3 above)
Comment (ALEGATOR1209) [reply src/test/kotlin/com/fictadvisor/pryomka/data/datasources/FsDocumentContentDataSourceTest.kt]: fixed

**Review by NeProgramist — APPROVED** [review]

**Pull request merged**